From bf3e453283ca1fd029f884bafa3563a9975cb62a Mon Sep 17 00:00:00 2001
From: Lorenzo Pirritano <lpirritano@anaconda.com>
Date: Tue, 23 Jan 2024 12:17:25 -0600
Subject: [PATCH] fix version number for setuptools

---
 streamlit_pandas_profiling/__init__.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/streamlit_pandas_profiling/__init__.py b/streamlit_pandas_profiling/__init__.py
index d418c8c..b0d6883 100644
--- a/streamlit_pandas_profiling/__init__.py
+++ b/streamlit_pandas_profiling/__init__.py
@@ -2,7 +2,9 @@ import streamlit as st
 
 from pathlib import Path
 from streamlit.components.v1.components import declare_component
-from streamlit_pandas_profiling.version import __release__, __version__
+from streamlit_pandas_profiling.version import __release__
+
+__version__ = "0.1.3"
 
 if __release__:
     _source = {"path": (Path(__file__).parent/"frontend"/"build").resolve()}
-- 
2.39.1

